src/wolfgame.py

```diff
@@ -6842,6 +6868,34 @@ def start(cli, nick, chan, forced = False, restart = ""):
             cli.msg(chan, "{0}: At most \u0002{1}\u0002 players may play.".format(nick, var.MAX_PLAYERS))
             return
 
+        if nick in var.START_VOTES:
+            cli.notice(nick, "You have already voted to start the game.")
+            return
+
+        start_votes_required = math.floor(var.start_votes_required(len(villagers)))
+        if len(var.START_VOTES) < start_votes_required:
+            # If there's only one more vote required, start the game immediately.
+            # Checked here to make sure that a player that has already voted can't
+            # vote again for the final start.
+            if len(var.START_VOTES) < start_votes_required - 1:
+                var.START_VOTES.add(nick)
+                msg = "{0} has voted to \u0002start\u0002 the game. \u0002{1}\u0002 more {2} required."
+                remaining_votes = start_votes_required - len(var.START_VOTES)
+
+                if remaining_votes == 1:
+                    cli.msg(chan, msg.format(nick, remaining_votes, 'vote'))
+                else:
+                    cli.msg(chan, msg.format(nick, remaining_votes, 'votes'))
+
+                # If this was the first vote
+                if len(var.START_VOTES) == 1:
+                    with var.WARNING_LOCK:
```

Comment:
lock in wrong place, this needs to be around anything that accesses var.START_VOTES (so add it in at line 6871 then indent all of this stuff by one)